withingsapp/migrations/0002_help_text.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2016-11-03 18:33
```

Comment:
@brad To be safe, I think we should generate the migrations with the minimum supported Django version (1.8). Higher versions of Django should be backwards compatible, but lower versions aren't always able to handle the migrations generated by the higher versions. https://docs.djangoproject.com/en/1.10/topics/migrations/#supporting-multiple-django-versions